@here/harp-materials/lib/SolidLineMaterial.ts

```diff
@@ -37,6 +37,28 @@ const DefinesLineCapsMapping: { [key: number]: LineCaps } = Object.keys(
     return r;
 }, ({} as any) as { [key: number]: LineCaps });
 
+export enum LineDashesModes {
+    DASHES_NONE = 0,
```

Comment:
I can not distinguish between DASHES_NONE and DASHES_SQUARE modes, I guess the first mode is just not needed.
See also my comments in the shader, regarding it.

Reply:
Removing it.